metadata/src/test/java/org/apache/kafka/controller/QuorumControllerIntegrationTestUtils.java

```diff
@@ -75,6 +77,27 @@ static BrokerRegistrationRequestData.FeatureCollection brokerFeatures(
         return features;
     }
 
+    /**
+     * Create a broker features collection for use in a registration request. MV and given features are included.
+     *
+     * @param minVersion    The minimum supported MV.
+     * @param maxVersion    The maximum supported MV.
+     * @param featureList   Other features to add.
+     */
+    static BrokerRegistrationRequestData.FeatureCollection brokerFeaturesPlusFeatureVersions(
```

Comment:
seems a bit awkward to use. can we just have a function parameter like a map from feature name to max version (and then assume that min versions are all 0?)

Reply:
Updated.